packages/@aws-cdk/lambda-layer-awscli/package.json

```diff
@@ -83,27 +84,33 @@
   },
   "dependencies": {
     "@aws-cdk/aws-lambda": "0.0.0",
+    "@aws-cdk/asset-awscli-v1": "2.2.0",
```

Comment:
This seems to be like it should be impossible since `@aws-cdk/asset-awscli-v1` depends on `aws-cdk-lib` in order to do `new Asset()`.